docroot/modules/custom/mass_content/mass_content.install

```diff
@@ -92,16 +92,6 @@ function mass_content_update_10001() {
   }
 }
 
-/**
```

Comment:
@arthurbaghdas as far as I can see, `mass_content_update_10003()` was released to production already, in some of the previous versions. 
I don't think we remove the previously executed hook_update_N, since it can lead to an unclear upgrade path. 
What was the reason removing the hook?

Reply:
I think you don't know that sometimes we do remove entire install files content as part of our code cleanup process. Don't understand what upgrade path do you refer to in this specific condition.

Reply:
Ok, that is new to me. It is contradicts with recommendation to not remove the `hook_update_N` if it was already executed on the destination database, to not mislead the other developers. 
But if it a common process on massgov then we should follow that rule.